libraries/rib-base/src/main/java/com/badoo/ribs/routing/state/action/single/Ext.kt

```diff
@@ -13,21 +13,22 @@ internal fun <C : Parcelable> Node<*>.createTransitionElements(
 ): List<TransitionElement<C>> {
     val elements = mutableListOf<TransitionElement<C>>()
 
-    val ribView = view
-    if (ribView != null) {
+    if (isViewless) {
+        children.forEach {
```

Comment:
Does it work in the following case? I have a view-less child with 2 view-full children. One of them in the backstack (it does not have attached view but not `isViewless` because view factory was provided to it) and the another one is visible and attached to the view. The first one will trigger `requireNotNull(ribView)` below, no? Or it works in some different way and I do not fully understand how nodes in the backstack behaves?

Reply:
Theoretically, no. Because if they are in the back stack and they stay there, they don't receive activation, so this code is not run. They receive activation when they're supposed to be attached to screen, and at this point the view is already created.

Sandbox app works fine with this change, but ui tests break related to Deactivate, which also uses this code block. Will investigate.

Reply:
It's definitely weird. I couldn't figure it out now, so I'll relax the check for now (same situation as before) so that at least the bugfix can be merged. Let's come back to it separately.